lib/tasks/database_updates/migrate_site_banners.rake

```diff
@@ -0,0 +1,15 @@
+namespace :database_updates do
+  task :migrate_site_banners => :environment do
+    # Update record_id's to use uuids instead of bigints
+    SiteBanner.find_each do |banner|
```

Comment:
Can we a little progress meter to this? Something that will report how far it's gone?
Maybe something like 
```suggestion
    total = SiteBanner.all.length
    SiteBanner.find_each.with_index do |banner, index|
      puts "#{index} / #{total} complete" if index % 5 == 0
```
And then one at the end

Reply:
Done ✔️